C#/Botkit/Microsoft.BotKit/BotWorker.cs

```diff
@@ -22,7 +21,7 @@ public class BotWorker
         /// </summary>
         /// <param name="controller">A pointer to the main Botkit controller</param>
         /// <param name="config">An object typically containing { dialogContext, reference, context, activity }</param>
-        public BotWorker(Botkit controller, object config)
+        public BotWorker(Core.Botkit controller, object config)
```

Comment:
`Core` in `Core.Botkit` can be removed
![image](https://user-images.githubusercontent.com/39467613/58023421-1f6fd400-7ae6-11e9-8f5f-5d8a6f3f2b63.png)

Reply:
Feedback applied